components/auth/src/main/java/org/cloudfoundry/credhub/config/auth/OAuth2AuthenticationExceptionHandler.java

```diff
@@ -6,6 +6,7 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import org.springframework.security.authentication.InsufficientAuthenticationException;
```

Comment:
This is not used.

Reply:
Unused import also causes `checkStyle` failure. I suggest doing `build -x test` locally at least to catch this kind of oversight.